src/main/main.iml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Again, we don't usually include Idea project files in the GitHub repos.

Reply:
removed